code/modules/power/port_gen.dm

```diff
@@ -170,6 +170,12 @@
 /obj/machinery/power/port_gen/pacman/proc/overheat()
 	explosion(src.loc, 2, 5, 2, -1)
 
+/obj/machinery/power/port_gen/pacman/MouseDrop_T(atom/dropping, mob/user)
+	if(is_type_in_typecache(dropping, sheet_path))
```

Comment:
You should be using istype() here since sheet_path is a typepath and not a list or cache.